src/main/kotlin/no/nav/familie/ef/sak/api/gui/PersonopplysningerController.kt

```diff
@@ -12,19 +19,33 @@ import org.springframework.web.bind.annotation.PostMapping
 import org.springframework.web.bind.annotation.RequestBody
 import org.springframework.web.bind.annotation.RequestMapping
 import org.springframework.web.bind.annotation.RestController
+import java.time.LocalDate
+import java.time.LocalDateTime
+import java.time.Month
+import java.util.*
 
 
 @RestController
 @RequestMapping(path = ["/api/personopplysninger"], produces = [APPLICATION_JSON_VALUE])
 @ProtectedWithClaims(issuer = "azuread")
 @Validated
 class PersonopplysningerController(private val personopplysningerService: PersonopplysningerService,
-                                   private val tilgangService: TilgangService) {
+                                   private val tilgangService: TilgangService,
+                                   private val behandlingService: BehandlingService,
+                                   private val fagsakService: FagsakService) {
 
     @PostMapping
     fun personopplysninger(@RequestBody personIdent: PersonIdentDto): Ressurs<PersonopplysningerDto> {
         tilgangService.validerTilgangTilPersonMedBarn(personIdent.personIdent)
         return Ressurs.success(personopplysningerService.hentPersonopplysninger(personIdent.personIdent))
     }
 
+    @PostMapping("/behandling")
+    fun personopplysninger(@RequestBody behandlingId: UUID): Ressurs<PersonopplysningerDto> {
```

Comment:
Noen som har forslag til hva denne path'en bør være? Kan bare ha et post-endepunkt på api/personopplysninger og vi ønsker i utgangspunktet ha et som tar i mot personIdent og et som tar behandlingId.

Reply:
POST på den ovenfor brukes i hovedsak fordi vi ikke skal logge fnr. 
Her ser jeg for meg at `@GetMapping("/behandling/{behandlingId}")` vil være en grei path og `@PathVariable("behandlingId") private behandlingId: UUID` kan fungere som input.

Reply:
Høres fornuftig ut :)